src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -313,15 +376,30 @@ public void LoadBytes()
         [Fact]
         public void ReflectionOnlyGetAssemblies()
         {
-            Assert.Equal(Array.Empty<Assembly>(), AppDomain.CurrentDomain.ReflectionOnlyGetAssemblies());
+            RemoteInvoke(() => {
+                Assert.Equal(0, AppDomain.CurrentDomain.ReflectionOnlyGetAssemblies().Length);
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void MonitoringIsEnabled()
         {
-            Assert.False(AppDomain.MonitoringIsEnabled);
-            AssertExtensions.Throws<ArgumentException>(null, () => {AppDomain.MonitoringIsEnabled = false;});
-            Assert.Throws<PlatformNotSupportedException>(() => {AppDomain.MonitoringIsEnabled = true;});
+            RemoteInvoke(() => {
+                Assert.False(AppDomain.MonitoringIsEnabled);
+                Assert.Throws<ArgumentException>(() => { AppDomain.MonitoringIsEnabled = false; });
+
+                if (PlatformDetection.IsFullFramework)
+                {
```

Comment:
Could you also check setting it to false here invokes an ArgumentException? https://referencesource.microsoft.com/#mscorlib/system/appdomain.cs,4770